example/cmd/device-simple/res/provisionwatchers/Simple-Provision-Watcher.yml

```diff
@@ -0,0 +1,17 @@
+name: Simple-Provision-Watcher
+labels:
+  - simple
+identifiers:
+  Address: simple[0-9]+
+  Port: 3[0-9]{2}
+blockingIdentifiers:
+  Port:
+    - '397'
+    - '398'
+    - '399'
```

Comment:
use double quote instead of single quote

Reply:
removed quote as it's not necessary for YAML